cli/azd/resources/scaffold/templates/host-containerapp.bicept

```diff
@@ -266,6 +235,44 @@ resource app 'Microsoft.App/containerApps@2023-05-02-preview' = {
     }
   }
 }
+{{- if .DbMySql}}
+
+resource linkerCreatorIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' = {
+  name: 'linkerCreatorIdentity'
+  location: location
+}
+
+resource linkerCreatorRole 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+  scope: resourceGroup()
+  name: guid(subscription().id, resourceGroup().id, linkerCreatorIdentity.id, 'linkerCreatorRole')
+  properties: {
+    roleDefinitionId:  subscriptionResourceId(
+      'Microsoft.Authorization/roleDefinitions', 'b24988ac-6180-42a0-ab88-20f7382dd24c')
```

Comment:
FYI: https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#general